src/app/js/formats/InvalidFormat.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import compose from 'recompose/compose';
+import translate from 'redux-polyglot/translate';
+import Warning from 'material-ui/svg-icons/alert/warning';
+
+import { fromUser } from '../sharedSelectors';
+import { polyglot as polyglotPropTypes } from '../propTypes';
+
+const capitalize = str =>
+    str ? str.charAt(0).toUpperCase() + str.slice(1) : '';
+
+const styles = {
```

Comment:
please use aphrodite/no-important, this way style can be overridden by theme css